LICENSE

```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
```

Comment:
Yes. Check [here](https://github.com/willianjusten/cards-jekyll-template/blob/master/LICENSE) and [here](https://github.com/willianjusten/cards-jekyll-template#license).